config/example/ir.yaml

```diff
@@ -48,6 +48,14 @@ morph:
         # TCP addresses in 'host:port' format
         - localhost:30333
         - localhost:30334
+      tls_config: # Additional addresses to listen to using TLS setup; must not overlap with `listen` section
```

Comment:
`tls` is OK, we have it this way in NeoGo since like forever (https://github.com/nspcc-dev/neo-go/commit/dfbb84ef381d9abf554ec60746c08747c7518170) but `tls` is easily recognizable (unlike SomeNeoProtocolParameters). Maybe it can be improved along with nspcc-dev/neo-go#2901.

Reply:
fixed